crates/compiler/test_syntax/tests/snapshots/pass/suffixed_multiple_defs.moduledefs.formatted.roc

```diff
@@ -1,4 +1,5 @@
 main =
     a! "Bar"
     x = B.b! "Foo"
+
```

Comment:
I am so happy to see this line 🎉 

I spent a while trying to fix this. Nice work.